front-end/src/views/CanteensPage.vue

```diff
@@ -0,0 +1,256 @@
+<template>
+  <div id="welcome-block">
+    <h1>
+      Découvrez les initiatives prises pour par nos cantines
+      pour une alimentation plus durable
+    </h1>
+    <img src="@/assets/desktop.svg" alt="">
+  </div>
+  <ul id="canteens-block">
+    <li v-for="canteen in canteens" :key="canteen.title" class="canteen-card">
+      <a class="location" :title="map[canteen.region].title" :href="map[canteen.region].href">
+        <img :alt="map[canteen.region].alt" :src="map[canteen.region].src">
+        <p class="attribution">{{ map[canteen.region].attribution }}</p>
+      </a>
+      <div class="summary">
+        <h2>{{ canteen.title }}</h2>
+        <ul class="statistics">
+          <li v-for="(statistic, key) in canteen.statistics" :key="key" class="statistic">
+            <div class="vertically-align-header">
+              <h3>{{ statisticTitle(key) }}</h3>
+            </div>
+            <p class="number">{{ statistic }}&nbsp;%</p>
+          </li>
```

Comment:
Je pense qu'ici on peut expliciter directement dans le html les 3 statistiques plutôt que d'aller les chercher dans une méthode (quitte à créer un mini-component si on veut éviter de dupliquer du html).

Reply:
Quel est le problème avec `v-for` ? J'ai remplacé le `statisticTitle` afin d'enlever une méthode qui est pas assez nécessaire.

Reply:
Dans ma tête l'utilisation d'une liste sous-tend que sa taille et son contenu peuvent changer, alors qu'ici c'est fixe, du coup ça me parait plus clair de l'expliciter. Ca donnerait quelque chose comme ça :
```
<li class="statistic">
  <div class="vertically-align-header">
    <h3>Produits bios</h3>
  </div>
  <p class="number">canteen.statistics['bio']</p>

  <div class="vertically-align-header">
    <h3>Produits durables</h3>
  </div>
  <p class="number">canteen.statistics['durable']</p>

  <div class="vertically-align-header">
    <h3>Produits issus du commerce équitable</h3>
  </div>
  <p class="number">canteen.statistics['equitable']</p>
</li>
 ```
 Et ensuite si on veut refactoriser on peut créer un component :
 ```
<template>
  <div class="vertically-align-header">
    <h3>{{title}}</h3>
  </div>
  <p class="number">{{stat}}</p>
</template>
```
Qui nous donnerait :
```
<li class="statistic">
  <StatComponent :title="'Produits bios'" :stat="canteen.statistics['bio']"/>
  <StatComponent :title="'Produits durables'" :stat="canteen.statistics['durable']"/>
  <StatComponent :title="'roduits issus du commerce équitable'" :stat="canteen.statistics['equitable']"/>
</li>
```

Reply:
J'ai fait le changement. Il me semble un peu lourd de créer un nouveau component pour ça, avec la gestion entre fichiers, mais je vois l'argument pour expliciter qu'on n'attente plus de trois numéros, alors j'ai pas un avis fort ici